model/game.rb

```diff
@@ -0,0 +1,27 @@
+require 'date'
+require_relative './item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_date, :publish_date
+
+  def initialize(multiplayer, last_played_date, publish_date)
```

Comment:
Hey @kaitcham . Good job so far. 
According to the requirement, **All Game class properties visible in the diagram should be defined and set up in the constructor method.**
published_date is not an attribute of `Game` class. Maybe you should not put it inside the constructor.